opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -345,15 +345,15 @@ class Translator {
      */
     _makeRequest(path, method, content, includeBearerHeader) {
        
-        var addBearerHeader = (typeof includeBearerHeader !== 'undefined') ?  includeBearerHeader : true;
+        includeBearerHeader = (typeof includeBearerHeader !== 'undefined') ?  includeBearerHeader : true;
```

Comment:
if you change the below to !!includeBearerHeader

---
In reply to: [89016237](https://github.com/openT2T/translators/pull/127#discussion_r89016237) [](ancestors = 89016237)